python/servo/package_commands.py

```diff
@@ -209,24 +209,21 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
             msi_path = path.join(dir_to_msi, "Servo.msi")
             print("Packaged Servo into {}".format(msi_path))
         else:
+            dir_to_temp = '/'.join(binary_path.split('/')[:-2]) + '/targz'
```

Comment:
Can you please replace all of the `.split('/')` and `'/'.join()` calls with `path.dirname` and `path.join`? All this string processing hurts my eyes :(